views/personal/name.pug

```diff
@@ -21,12 +22,10 @@ block content
         tr
           td
             div #{data.personal.firstName} #{data.personal.lastName}
+  
+  form.cra-form(method='post')
+    +yesNoRadios('name', null, 'Is this your name?', errors)
 
-  div
-    details.
-      <summary><span>#{__('Help with name confirmation')}</span></summary>
-      <p>#{__('If you have changed your name since the last time you filed your taxes,')} <a href=#{__('https://www.canada.ca/en/revenue-agency/services/tax/individuals/frequently-asked-questions-individuals/should-you-tell-cra-about-your-change-name.html')} target="_blank">#{__('get in contact with the CRA to update your information')}</a>.</p>
-      <p>#{__('Once updated, you can proceed to use this service.')}</p>
```

Comment:
Should we link to any of the CRA guidance around name changes on the offramp page?

Reply:
Also, on the `/financial/income` page, we have a revealing element that shows up if you're about to say "no". Do we need that here, @brianhendrick1?

Reply:
Brian and I had talked about this a little bit, and I think we're going to leave it in for now until we think more on it.